.vscode/cspell.json

```diff
@@ -81,7 +81,10 @@
     "gmock",
     "gtest",
     "Gzrs",
+    "hcertstore",
```

Comment:
You might consider putting these in a local cspell.json file rather than the global one. Not a huge deal, but something to consider if the words aren't likely to be broadly used.

Reply:
Done. All new words except for `HLOCAL` were moved to a local `cspell.json` file.